sdk/keyvault/keyvault-keys/src/transformations.ts

```diff
@@ -0,0 +1,55 @@
+import { DeletedKeyBundle, KeyBundle } from "./generated/models";
+import { parseKeyVaultKeyId } from "./identifier";
+import { DeletedKey, KeyVaultKey, JsonWebKey, KeyOperation } from "./keysModels";
+
+/**
+ * @internal
+ * @ignore
+ * Shapes the exposed {@link KeyVaultKey} based on either a received key bundle or deleted key bundle.
+ */
+export function getKeyFromKeyBundle(bundle: KeyBundle | DeletedKeyBundle): KeyVaultKey {
+  const keyBundle = bundle as KeyBundle;
+  const deletedKeyBundle = bundle as DeletedKeyBundle;
```

Comment:
Feels a little weird to work with the same object in two different variables with two different interfaces.

Reply:
This transformation was already there, but I agree!